.github/sync-repo-settings.yaml

```diff
@@ -197,6 +197,26 @@ branchProtectionRules:
       - conformance
       - library_generation
       - unmanaged_dependency_check
+  - pattern: 2.60.x
+    isAdminEnforced: true
+    requiredApprovingReviewCount: 1
+    requiresCodeOwnerReviews: true
+    requiresStrictStatusChecks: false
+    requiredStatusCheckContexts:
+      - dependencies (17)
+      - lint
+      - clirr
+      - units (8)
+      - units (11)
```

Comment:
Do we need to add units (17) and units (21) too? https://github.com/googleapis/java-bigtable/blob/95e7abf36595e9e0b99023bcc21ae4893d7df459/.github/workflows/ci.yaml#L28